torch/csrc/distributed/rpc/init.cpp

```diff
@@ -284,6 +286,46 @@ If the future completes with an error, an exception is thrown.
         return pyRpcPythonUdf(agent, dst, pickledPythonUDF, tensors);
       });
 
+  struct PythonFutureWrapper {
```

Comment:
can you add a TODO here saying this should be properly merged with the jit PythonFutureWrapper as one of the task when merging these two future implementations?

Reply:
will do

Reply:
Would be nice to create a gh issue for this follow up too.